sdk/nginx/Azure.ResourceManager.Nginx/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 1.1.0 (2024-10-11)
+## 1.1.0 (2024-10-23)
```

Comment:
Since the spec is in a preview version, the corresponding SDK is available only as a beta release, not a stable one.